reports/2025-04-04-Q1-report.md

```diff
@@ -0,0 +1,114 @@
+# Haskell Security Response Team - 2025 January–March report
+
+The Haskell Security Response Team (SRT) is a volunteer organization
+within the Haskell Foundation that is building tools and processes
+to aid the entire Haskell ecosystem in assessing and responding to
+security risks.  In particular, we maintain a [database][repo] of
+security advisories that can serve as a data source for security
+tooling.
+
+This report details the SRT activities for January–March 2025 (a
+two month period due to extension of the previous reporting period).
```

Comment:
it's a bit ambiguous

Reply:
Thanks, it was a copy/paste error from the Q4 report :sweat_smile: